test/test_helper.exs

```diff
@@ -41,7 +41,7 @@ Livebook.Storage.current().insert(:settings, "global", autosave_path: nil)
 
 erl_docs_available? = Code.fetch_docs(:gen_server) != {:error, :chunk_not_found}
 
-exclude = []
-exclude = if erl_docs_available?, do: exclude, else: Keyword.put(exclude, :erl_docs, true)
-
-ExUnit.start(assert_receive_timeout: 1_500, exclude: exclude)
+ExUnit.start(
+  assert_receive_timeout: 1_500,
+  exclude: [erl_docs: erl_docs_available?, enterprise_integration: true]
```

Comment:
We should check if there is something at the enterprise `app_dir` and, if so, set it to true. Otherwise, set it to false!